package.json

```diff
@@ -66,6 +69,9 @@
     "rollup-plugin-postcss": "4.0.0",
     "rollup-plugin-styles": "3.14.1",
     "rollup-plugin-typescript2": "0.30.0",
+    "sass": "1.43.4",
+    "sass-loader": "10.1.1",
+    "style-loader": "2.0.0",
     "stylelint": "14.0.1",
     "stylelint-config-standard": "23.0.0",
     "ts-jest": "27.0.4",
```

Comment:
C'est voulu que la majorité des versions soient set in stone plutôt que ^ ?

Reply:
oui car les versions plus récente engendre de problème de build avec Storybook.